localisation/00_wc_sad.csv

```diff
@@ -774,3 +774,10 @@ forge_hammer_of_the_twilight;Forge the Hammer of Twilight;;Schmiedet den Schatte
 forge_hammer_of_the_twilight_desc;Forge an elemential hammer imbued with the power of the Old Gods in image of the Twilight's Hammer Cult symbol.;;Schmiedet einen Elementhammer mit der Kraft der alten G�tter als Symbol des Kults des Zwielichthammers.;;;;;;;;;;;x
 forge_hammer_of_the_twilight_power_desc;Enables the decision to forge the Hammer of Twilight.;;Erm�glicht die Entscheidung, den Schattenhammer zu schmieden.;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
+open_ulduar;Ulduar Expedition;Ulduar Expedition;;;;;;;;;;;;x
+open_ulduar_desc;Prepare an expedition with goal of discovering the the forgotten Titan city of Ulduar.;;;;;;;;;;;;;x
+EVTDESC_WCUTF.10;You have contracted a famous explorer to lead an expedition into frozen wastes of boreal Northrend in search of Titan city of Ulduar. Many believe this place exists only in legends and that all you can find in the north is glaciers and savage giants but your explorer thinks differently. He is well prepared and assures you that the secrets of Titan power will soon be yours.;;;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCUTF.10;You have contracted a famous explorer to lead an expedition into the frozen wastes of boreal Northrend in search of the Titan city of Ulduar. Many believe this place exists only in legends and that all you will find in the north is glaciers and savage giants, but your explorer thinks otherwise. He is well prepared and assures you that the secrets of Titan power will soon be yours.;;;;;;;;;;;;;x
```